protoclass/detection/extract_lbp_volume_mssp.m

```diff
@@ -0,0 +1,230 @@
+function [ feature_mat_vol ] = extract_lbp_volume_mssp( in_vol, pyr_num_lev, NumNeighbors, Radius, CellSize, MODE, Mapping )
+% EXTRACT_LBP_VOLUME_MSSP Function to extract LBP descriptor from a
+% volume using a pyramidal and spatial blocks within a pyramids (SP) approach
+% MSSP method in comparison to SP method extract spatial blocks from the
+% image and also from the intersections of the blocks
+%     [ feature_mat_vol ] = extract_lbp_volume_mssp( in_vol,
+%     pyr_num_lev, NumNeighbors, Radius, CellSize, Mapping, MODE ) 
+%
+% Required arguments:
+%     in_vol : 3D array
+%         Entire volume.
+%
+%     pyr_num_lev : int
+%         The number of level in the pyramid.
+%
+%     NumNeighbors : int
+%         Number of neighbors.
+%
+%     Radius : int
+%         Radius of circular pattern to select neighbors.
+%
+%     CellSize : vector
+%          Cell size.
+%   
+%     MODE: string 
+%          histogram ('h') , normalized histogram ('nh') 
+%
+%     Mapping: string
```

Comment:
I would probably put `mapping` instead of `Mapping`. It depends of the LBP documentation.